_posts/2020-08-03-roadmap.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Roadmap
+---
+
+On June 18 2020, MDAnalysis was pleased to release the first major version, 1.0.0. As described in our [2019 roadmap](https://www.mdanalysis.org/2019/11/06/roadmap/), this is the last version that supports 2.7. We will continue backporting relevant bug fixes where feasible (e.g. the upcoming 1.0.1), but the next major release will be 2.0.0, which will support Python 3.6+.
```

Comment:
Create the link as
```
[2019 roadmap]({{ site.baseurl }}{% post_url 2019-11-06-roadmap %})
```
so that local building also works and generally let jekyll worry about linking.